index.html

```diff
@@ -491,7 +491,7 @@ <h2 about="#introduction" property="schema:name" typeof="deo:Introduction">Intro
             <div datatype="rdf:HTML" property="schema:description">
               <p><em>This section is non-normative.</em></p>
 
-              <p id="motivation" rel="schema:hasPart" resource="#motivation" typeof="deo:Motivation"><span datatype="rdf:HTML" property="schema:description">Unauthorized access or modification of Web resources could lead to unintended loss, damage or disclosure of data. To support read-write operations within the framework of HTTP, mechanisms for expressing and application of authorization conditions are needed to meet the social requirements of decentralised applications.</span></p>
+              <p id="motivation" rel="schema:hasPart" resource="#motivation" typeof="deo:Motivation"><span datatype="rdf:HTML" property="schema:description">Unauthorized access or modification of Web resources could lead to unintended loss, damage or disclosure of data. To support read-write operations within the framework of HTTP, mechanisms for expressing and applying authorization conditions are needed to meet the social requirements of decentralised applications.</span></p>
```

Comment:
Merged after 2 days... Should have been 3.
```suggestion
              <p id="motivation" rel="schema:hasPart" resource="#motivation" typeof="deo:Motivation"><span datatype="rdf:HTML" property="schema:description">Unauthorized access or modification of Web resources could lead to unintended loss, damage or disclosure of data. To support read-write operations within the framework of HTTP and meet the social requirements of decentralised applications, mechanisms for expressing and applying authorization conditions are needed.</span></p>
```

Reply:
Thanks Ted. Erich's suggestion was a fine minor editorial improvement that I believe I can merge right away. I've also included your suggestion in commit https://github.com/solid/web-access-control-spec/commit/4e331ac6e65f3a06d09b47e6d23a082a14e9d9ad .